src/steadystateproblem.cpp

```diff
@@ -49,6 +49,14 @@ SteadystateProblem::SteadystateProblem(const Solver &solver, Model &model)
         throw AmiException("Preequilibration using adjoint sensitivities "
                            "is not compatible with using forward "
                            "sensitivities during simulation");
+
+    /* Turn off Newton's method if SteadyStateSensitivityMode is integrationOnly 
+    in forward sensitivity case */
+    if (solver.getSensitivityMethod() == SensitivityMethod::forward && 
```

Comment:
I would prefer if this was rather implemented as a check in `findSteadyState`. On the one hand this means we do not apply any changes to the solver that the user may not expect (changing newton steps means switching to a different sensitivity mode afterwards wouldn't necessarily reactivate it). On the other hand it avoids issues in edge cases when convergence is achieved in the first newton solver step, which is carried out even when newton steps are set to 0.

Reply:
Ah, I see. Wasn't sure about this part. Thanks for the explanation